uktrade/great-cms
# KLS-1921 Validation for closed Events

**Description**

This ticket adds validation around Event completion. An event that is marked as completed should return an error on save (error message showing on UI) if the aforementioned event has not been marked as closed for further bookings.

**To Test**

Go into the great-cms admin --> select and event --> mark as 'complete' (whilst ensuring it has NOT been marked as 'closed for bookings' --> click save --> A validation error should appear. 

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-1921

### Reviewing help

- 
### Merging
